sonic_platform_base/sonic_xcvr/api/public/sff8636.py

```diff
@@ -403,3 +405,46 @@ def get_lpmode(self):
         # Since typically optics come up by default set to high power, in this case,
         # power_override not being set, function will return high power mode.
         return power_set and power_override
+
+    def get_power_class(self):
+        '''
+        Retrieves power class of the module.
+
+        Returns:
+            int: Power class of the module, None if it fails
+        '''
+        power_class_str = self.xcvr_eeprom.read(consts.POWER_CLASS_FIELD)
+        power_class = None
+
+        if power_class_str is None:
+            return power_class
+
+        pattern = re.compile(self.POWER_CLASS_PATTERN)
+        match = pattern.match(power_class_str)
+        if match:
+            power_class = int(match.group(1))
+
+        return power_class
+
+    def set_high_power_class(self, power_class, enable):
+        '''
+        This function sets high power class for the module.
+        It is only applicable for power class >= 5.
+
+        Args:
+            power_class (int): Power class to be enabled/disabled
+            enable (bool): True to enable high power class, False to disable
+
+        Returns:
+            bool: True if the provision succeeds, False if it fails
+        '''
+        ret = True
+
+        if power_class < 5:
+            return ret
+        elif power_class >= 8:
+            ret = self.xcvr_eeprom.write(consts.HIGH_POWER_CLASS_ENABLE_CLASS_8, enable)
+        else:  # Power class 5, 6, 7
+            ret = self.xcvr_eeprom.write(consts.HIGH_POWER_CLASS_ENABLE_CLASS_5_TO_7, enable)
```

Comment:
@longhuan-cisco can we simply this as follows
if power_class >= 8:

else if power_class >= 5

Reply:
updated it.